README.md

```diff
@@ -244,6 +271,63 @@ contains settings for ECS managed health checks.
 * `alias` – (Optional) An [alias](#alias) block used to define a Route 53 alias record
 that points to the load balancer. Requires that a `load_balancer` block is definied.
 
+* `autoscale` – (Optional) An [autoscale](#autoscale) block used to create an autoscaling configuration. Requires that a `autoscale` block is defined.
+
+`autoscale`
+----------
+
+An `autoscale` block supports the following for autoscale up and down policies:
+
+* `autoscale_scale_up` - (Optional) The name of the autoscaling policy for scale up. Default name is ('service name' up) 
```

Comment:
"${var.name}-down"
"${var.name}-up"

(If you agree to the renaming posed in a later comment)